libexec/alluxio-config.sh

```diff
@@ -31,6 +31,7 @@ VERSION=2.3.0-SNAPSHOT
 ALLUXIO_HOME=$(dirname $(dirname "${this}"))
 ALLUXIO_ASSEMBLY_CLIENT_JAR="${ALLUXIO_HOME}/assembly/client/target/alluxio-assembly-client-${VERSION}-jar-with-dependencies.jar"
 ALLUXIO_ASSEMBLY_SERVER_JAR="${ALLUXIO_HOME}/assembly/server/target/alluxio-assembly-server-${VERSION}-jar-with-dependencies.jar"
+ALLUXIO_VALIDATION_TOOL_JAR="${ALLUXIO_HOME}/lib/alluxio-integration-tools-validation-${VERSION}.jar"
```

Comment:
@LuQQiu @madanadit I have to add the validation tool jar to a separate classpath, as now we have `runUfsTests` and `validateEnv` to directly invoke from there. PTAL if this is acceptable. Thanks!

Reply:
why can you not add the validation jar to the client classpath itself?

Reply:
that sounds better, add the `ALLUXIO_VALIDATION_TOOL_JAR` to the `ALLUXIO_CLIENT_CLASSPATH` (although i am not 100% it works, maybe try it out)

Reply:
Adding the validation jar to the client classpath works. By using this separate path i was trying to minimize the impact on client classpath. Since that's fine I'm adding validation jar to the client path.